index.html

```diff
@@ -6,7 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="styles.css">
   <script src="data.js"></script>
-  <script src="index.js"></script>
+  <script src="index.js" defer></script>
```

Comment:
Yup! Another way to handle this btw, is to move the script tag down to right above the `</body>` tag in your HTML file. 

And soon, we'll learn ANOTHERRRR way to handle this.